digital_land/repository/entry_repository.py

```diff
@@ -124,9 +127,6 @@ def _create_schema(self):
                 entity TEXT,
                 attribute TEXT,
                 value TEXT,
-                entry_date,
-                start_date,
```

Comment:
Why were these removed?

Reply:
we currently don't have a meaning for them. Can add them again easily if required.